src/services/teams/hooks/index.js

```diff
@@ -515,9 +518,32 @@ const testChangesForPermissionRouting = globalHooks.ifNotLocal(async (hook) => {
 			throw new Forbidden('Can not adding users from other schools.');
 		}
 		if (isLeaveTeam) {
```

Comment:
for me it seems that isLeaveTeam will not be true if another user is trying to remove a teamowner

Reply:
should it? If someone removes a user the user is not leaving